lib/renderer.js

```diff
@@ -1116,6 +1116,26 @@
         
         return component.document.layers.visit(layerHasEffects);
     };
+
+    /**
+     * Check if the document returns empty bounds.
+     * 
+     * @private
+     * @param {Document} document incoming document where empty layers will be searched for.
+     * @return {boolean}
+     */
+    PixmapRenderer.prototype._documentHasZeroBounds = function (document) {
+
+        var docBounds = this._getContentBounds(document.layers, true);
+        if (docBounds.top === 0 &&
+            docBounds.left === 0 &&
+            docBounds.bottom === 0 &&
+            docBounds.right === 0) {
```

Comment:
This is to check to see if we need to ask for the final image bounds and not rely on the quick ones when we got the documentInfo (there are cases in getDOcumentInfo where the bounds are not correct because it's too expensive to calculate if they aren't getting used). If there is something outside of the document, then we know that it was calculated correctly. We know that if a layer is `0,0,0,0` specifically there is a good chance that it's not correct and we need to ask for the specific bounds